### ISSUE TITLE
Tee/once file does not support tilde for home directory

### ISSUE THREAD

**Issue by tsroten** [dbcli/mycli#467]
If you try to write to a tee file or one-time output file using the `~` character for the home directory, it does not work:

```
mysql root@localhost:(none)> tee ~/test.csv
Cannot write to file '~/test.csv': No such file or directory
```

Instead we should make it easy for the user to type paths in their home directory.

**Comment by meeuw** [comment]
Right, we should use `os.path.expanduser` for that, this should also be added to the other commands that take a filename.

**Comment by tsroten** [comment]
This is now fixed as of [version 1.11.0](https://pypi.org/project/mycli/1.11.0/).
